Python/crossfire/crossfire/client.py

```diff
@@ -115,3 +115,17 @@ def get(self, *args, **kwargs):
 
     def states(self, format=None):
         return self.get(f"{URL}/states", format=format)
+
+    def cities(self, **kwargs):
+        city_url = f"{URL}/cities"
+        format = kwargs.pop("format", None)
+        city_id = kwargs.pop("city_id", None)
+        city_name = kwargs.pop("city_name", None)
+        state_id = kwargs.pop("state_id", None)
+        if city_id:
+            city_url = f"{city_url}?cityId={city_id}"
+        elif city_name:
+            city_url = f"{city_url}?cityName={city_name}"
+        elif state_id:
+            city_url = f"{city_url}?stateId={state_id}"
+        return self.get(city_url, format=format)
```

Comment:
Não é seguro apenas juntar _string_ para formar parâmetros de URL com valores que vem de quem usa a API. O ideal é usar [uma ferramenta própria para isso](https://docs.python.org/3/library/urllib.parse.html#urllib.parse.urlencode).

```suggestion
        params = {"cityId": uuid, "cityName": name, "stateId": state_uuid}
        cleaned = urlencode({key: value for key, value in params.items() if value})
        return self.get(f"{URL}/cities?{cleaned}", format=format)
```

No mais, fico com a impressão que tua função teria um bug com mais de um parâmetro de filtro (URL inválida como `/cities/?name=FourtyTwo?uuid=42` ao invés de `/cities/?name=FourtyTwo&uuid=42`).